### PR TITLE
Add information about hotels in Cardiff

### PR DESCRIPTION
**Pull request by alexwlchan**
Most of the text is copied wholesale from the PyCon UK 2016 site, with just the information about which hotels are offering us special rates being rearranged slightly.

### CODE REVIEW

**Comment by alexwlchan** [comment]
Okay, so this is breaking because http-crawler doesn’t know about mailto: links. Guess it’s time for me to finally fix that…

Related: https://github.com/inglesp/http-crawler/issues/6

**Comment by alexwlchan** [comment]
While I prepare a patch for that, can I get another pair of eyes on the content please?

File: pages/accommodation.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+title: Accommodation
+---
+
+# Hotels
+
+## Recommended partner hotels
+
+The hotels listed in this section have given us preferential rates or terms or both. Their allocation of rooms for
+PyCon UK attendees is likely to be exhausted well before the conference, so **you are strongly advised to book as early
+as you can**.
```
Comment (inglesp) [inline pages/accommodation.md]: There's only one hotel in this list!

File: pages/accommodation.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+they're gone.
+
+#### Sleeperz Hotel
+
+[Sleeperz Hotel](http://www.sleeperz.com) (tel +44 2920 478747); [email Sleeperz Hotel
+reservations](mailto:reservations@sleeperzcardiff.com)) is a comfortable budget hotel, a minute's walk from the station and 15
+minutes' walk from City Hall.
+
+Please quote the *PYCON SELECT* rate when booking (phone or email). The negotiated PyCon UK rates include breakfast, in a
+double or twin room.
```
Comment (inglesp) [inline pages/accommodation.md]: Do we know what the negotiated rates are?

File: pages/accommodation.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+#### Sandringham Hotel
+
+The [Sandringham Hotel](http://sandringham-hotel.com) (tel +44 2920 232161); [email Sandringham Hotel
+reservations](mailto:mm@sandringham-hotel.com)) is mid-way between City Hall (about 10
+minutes' walk) and Cardiff Central Railway station.
+
+Please note that the hotel has five floors **but does not have a lift**. It's **not** suitable for people with reduced
+mobility.
+
+Please use the code *trip1* when booking (phone or email).
```
Comment (inglesp) [inline pages/accommodation.md]: Presumably this line is a hangover from last year.

File: pages/accommodation.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+## Use booking sites
+
+Check sites such as [Hotels.com](http://hotels.com) and [Booking.com](http://booking.com) for other hotels. (Apparently
+if you put your browser into private or incognito mode, you can sometimes get different, better, prices. Who knows?)
+
+Some hotels don’t even appear on those sites, never mind offer discounts to Pythonistas; you need to go to their
+websites directly and pay the same as everyone else:
+
+#### Travelodge Hotels
```
Comment (inglesp) [inline pages/accommodation.md]: Should these, and the Premier Inn, be listed under "Other Hotels"?

**Review by inglesp — CHANGES_REQUESTED** [review]

File: pages/accommodation.md (hunk 2 above)
Comment (alexwlchan) [reply pages/accommodation.md]: I don’t, no.

File: pages/accommodation.md (hunk 1 above)
Comment (alexwlchan) [reply pages/accommodation.md]: Oops. Plurals are hard.

File: pages/accommodation.md (hunk 3 above)
Comment (alexwlchan) [reply pages/accommodation.md]: Mmmm. I stayed there last year, and this was a generic voucher (although it is a hangover from last year). I’ll remove it for now.

File: pages/accommodation.md (hunk 4 above)
Comment (alexwlchan) [reply pages/accommodation.md]: Yep, think that’s a holdover from last year.

**Comment by alexwlchan** [comment]
@inglesp I’ve addressed your comments and rebased against current master; anything else?

**Comment by ntoll** [comment]
Please see #52 WRT information missing from this PR.

**Pull request merged**